### PR TITLE
Clearification

### PR DESCRIPTION
**Pull request by poi33**
Small addition to avoid confusion when trying out the starter and app.

### CODE REVIEW

File: docs/index.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ include::variables.adoc[]
 
 == Introduction
 
-Explorer is an enterprise search application that supports collecting, and searching across data from different sources. 
+Explorer is an enterprise search application that supports collecting, and searching across data from different sources. You need a license to run the application.
```
Comment (ComLock) [inline docs/index.adoc:8]: You do not need a license to run the application. You need a license to have more than 3 collections, use synonyms or more than 2 interfaces.

Comment (ComLock) [reply docs/index.adoc:8]: https://github.com/enonic/app-explorer/blob/master/src/main/resources/assets/react/Collections.jsx#L162

Comment (ComLock) [reply docs/index.adoc:8]: https://github.com/enonic/app-explorer/blob/master/src/main/resources/assets/react/thesaurus/NewOrEditThesaurus.jsx#L65

Comment (ComLock) [reply docs/index.adoc:8]: You are allowed 2 interfaces https://github.com/enonic/app-explorer/blob/master/src/main/resources/assets/react/interfaces/NewOrEditInterfaceModal.jsx#L63

**Review by ComLock — CHANGES_REQUESTED** [review]
See comments about license

**Comment by poi33** [comment]
Okay my bad, then we can close this.

**Pull request closed without merging**